src/context/AuthContext.tsx

```diff
@@ -65,24 +67,55 @@ export const AuthProvider: FC<{ children: ReactNode }> = ({ children }) => {
             .catch((error) => {
                 throw new Error('Error during logout:', error);
             })
+        // return await makeRequest('POST', `logout`, {})
```

Comment:
jak robimy PRke to takie komentarze juz powinny byc usuniete

Reply:
zrozumiano :)